gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -1,91 +1,151 @@
-= Anypoint Platform Gateways
+= Flex Gateway Overview
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-An API gateway allows you to add a dedicated orchestration layer on top of your backend APIs and services
-to help you separate orchestration from implementation concerns. You can then leverage the governance capabilities
-of API Manager to apply, among other capabilities, throttling, security, caching, and logging to your APIs.
+Anypoint Flex Gateway is an Envoy-based, ultrafast lightweight API gateway designed to manage and secure APIs running anywhere. Built to seamlessly integrate with DevOps and CI/CD workflows, Flex Gateway delivers the performance required for the most demanding applications and microservices while providing enterprise security and manageability across any environment.
 
-Anypoint Platform offers three different runtime options for managing and securing your APIs.
+== Flex Gateway Architecture
 
-== Anypoint Flex Gateway
+Flex Gateway contains two components that work in tandem:
 
-Anypoint Flex Gateway is an ultrafast API gateway designed to manage and secure APIs running anywhere.
-Built to seamlessly integrate with DevOps and CI/CD workflows, Anypoint Flex Gateway delivers the
-performance required for the most demanding applications and microservices while providing enterprise
-security and manageability across any environment.
+* The control plane
```

Comment:
Remove the articles

```
* Control plane 
* Runtime
```

Reply:
Do these need more info - like descriptions of each? Oh nm, I see the graphic.

Reply:
The graphic is quite small and hard to read.

Reply:
Fixed.

Also increased the font sizes in the graphic.